ntbs-service-unit-tests/DataMigration/NotificationMapperTest.cs

```diff
@@ -271,7 +271,6 @@ public async Task correctlyCreates_PostMortemNotification()
             // Assert
             Assert.Single(notification.TreatmentEvents);
             Assert.True(notification.ClinicalDetails.IsPostMortem);
-            Assert.True(notification.ShouldBeClosed());
```

Comment:
Removed because the check that the notification is mapped as closed is already here so checking ShouldBeClosed too doesn't make much sense (and with the new definition isn't true).